liquibase-standard/src/main/java/liquibase/change/core/RawSQLChange.java

```diff
@@ -23,7 +24,15 @@ public class RawSQLChange extends AbstractSQLChange {
     private String comment;
 
     private Boolean rerunnable;
-    
+
+    @Getter
+    @Setter
+    private Integer sqlStartLine;
+
+    @Getter
+    @Setter
+    private Integer sqlEndLine;
```

Comment:
Will this influence the way checksums are calculated?

Reply:
No, the attributes aren't used in the calculation, just the generated SQL for the change.